packages/vitest/src/node/plugins/cssEnabler.ts

```diff
@@ -9,10 +10,14 @@ export function CSSEnablerPlugin(ctx: Vitest): VitePlugin {
   const shouldProcess = (id: string) => {
     if (!isCss(id))
       return true
-    const { processCss } = ctx.config
-    if (typeof processCss === 'boolean')
-      return processCss
-    return processCss.some(re => re.test(id))
+    const { css } = ctx.config
+    if (typeof css === 'boolean')
+      return css
+    if (toArray(css.exclude).some(re => re.test(id)))
+      return false
+    if (toArray(css.include).some(re => re.test(id)))
+      return true
+    return true
```

Comment:
I know there is a utility like this, and I don't know why we don't use it. When doing this PR, I also haven't found it, so I didn't install it.

TBH I don't see any real value in using it here. If we want to install utils, we might as well change it's usages everywhere.

Reply:
Also, it's won't really work here. I need to return `true`, if the file is not inside `include` and `exclude`

Reply:
Yes, probably better for another PR to do this for all include/exclude like options. The utils are already used in Vite core so it isn't adding anything new. I actually wonder if we shouldn't expose it from Vite as a util so more people in the ecosystem will use it. I think I'll propose that in a PR in vite.

Reply:
For reference: 
- https://github.com/vitejs/vite/pull/8562

Reply:
for deps' include / exclude is different from `createFilter` semantically (as `['vue']` stands for the `vue` package instead of glob). For `css` I think sure we could use it.